src/handlers/get_messages.rs

```diff
@@ -101,6 +101,13 @@ pub async fn handler(
     increment_counter!(state.metrics, get_queries);
     increment_counter_with!(state.metrics, served_items, messages.len() as u64);
 
+    // Prematurely make breaking change of removing method
+    // https://walletconnect.slack.com/archives/C04CKNV4GN8/p1688127376863359
```

Comment:
Nit: I wouldn't put slack links into code

Reply:
What is your thinking? I figured good as a reference/context

Reply:
Add it to the PR description then, not the code.